packages/react-noop-renderer/package.json

```diff
@@ -10,7 +10,8 @@
     "fbjs": "^0.8.16",
     "object-assign": "^4.1.1",
     "prop-types": "^15.6.0",
-    "regenerator-runtime": "^0.11.0"
+    "regenerator-runtime": "^0.11.0",
+    "react-reconciler": "^0.6.0"
```

Comment:
Our release script probably won't know to bump it, so eventually the local version will stop satisfying it. Can we use `"*"` as a version instead? Since we never publish it, that should be enough as a workaround.